QuickDeployment/QuickDeploymentSetup.md

```diff
@@ -5,113 +5,124 @@ You will receive a download link via email for an archive of the VM image. Once
+        - [Change the database access credentials](#change-the-database-access-credentials)
+        - [Reinstall the chocolatey-management-service package](#reinstall-the-chocolatey-management-service-package)
+        - [Reinstall the chocolatey-management-web package](#reinstall-the-chocolatey-management-web-package)
 
 <!-- /TOC -->
 
 ## Setup Considerations
 
 The following are points to keep in mind during initial setup:
 
-* You will need access to AWS to download the environment (specifically `s3.amazonaws.com`).
-* Hostname/FQDN changes will invalidate all scripts and certificates. Thus, if you plan to change the hostname, you must do so prior to running any setup scripts. If you run into issues, refer to the README file on the desktop of the VM.
-* Self-signed certificates are generated by default. If you plan to use your own certificates, please reach out to Support for assistance.
-* The back-end database is configured with no outbound connections - if you plan to change this, please reach out to Support for assistance.
-* If you intend to use Nexus outside of your corporate network without the use of a VPN, you will be required to configure RBAC on the repositories housed inside of the repository server. This is to ensure that the packages stored on the server are not publicly accessible without authentication.
+- You will need access to AWS to download the environment (specifically `s3.amazonaws.com`).
+- Hostname/FQDN changes will invalidate all scripts and certificates.
+  Thus, if you plan to change the hostname, you must do so prior to running any setup scripts.
+  If you run into issues, refer to the README file on the desktop of the VM.
+- Self-signed certificates are generated by default.
+  If you plan to use your own certificates, please reach out to Support for assistance.
+- The back-end database is configured with no outbound connections - if you plan to change this, please reach out to Support for assistance.
+- If you intend to use Nexus outside of your corporate network without the use of a VPN, you will be required to configure RBAC on the repositories housed inside of the repository server.
+  This is to ensure that the packages stored on the server are not publicly accessible without authentication.
 
 ## Import Virtual Environment
 
 ### Platform: Hyper-V (Appliance)
 
-1. Download zip archive containing the Hyper-V VM directory structure, and unzip it to the directory you wish to store it in
-
+1. Download zip archive containing the Hyper-V VM directory structure, and unzip it to the directory you wish to store it in.
 2. Open Hyper-V Manager, and select `Import Virtual Machine` from the right sidebar (or  Action menu), and choose the folder you extracted from the above archive (e.g. ChocoServer).
-
 3. Increase the size of the VHD, for example, to 100GB. Increase to what you feel comfortable with.
 
 ```powershell
 Resize-VHD -Path C:\path\to\QuickDeploy Environment.vhd -SizeInBytes 100GB
```

Comment:
Should this include the Script bypass?

Reply:
They're copy-pasting a cmdlet call, so it's not needed, no scripts are getting run.

They _might_ hit an issue if this is a script function (not sure there; perhaps @steviecoaster can confirm if it's a function or a cmdlet) I suppose?

Reply:
No idea, just more for consistency in here.